samples/wifi/scan/src/main.c

```diff
@@ -43,6 +43,12 @@ LOG_MODULE_REGISTER(scan, CONFIG_LOG_DEFAULT_LEVEL);
 
 static uint32_t scan_result;
 
+const struct wifi_scan_params tests[] = {
```

Comment:
Samples are to demonstrate how to use API, should only have primary usecases with inputs from Kconfig, all combinations/tests should be under `tests` usign Ztest.

Reply:
As discussed offline lets do this in the next phase.